locale/en/blog/announcements/interactive-2016-north-america-schedule.md

```diff
@@ -7,13 +7,13 @@ slug: interactive-2016-north-america-schedule
 layout: blog-post.hbs
 ---
 
-> IBM, Netflix, Microsoft and leading community experts to showcase the current and future of Node.js
+> IBM, Netflix, Microsoft, and leading community experts to showcase the current and future of Node.js
```

Comment:
@fene I agree that it is debatable. I added them because few places in this document use them. I thought it would be better if we consistently used them.